test/integration/app/spec_build.js

```diff
@@ -315,6 +315,7 @@ describe('Spec option and build integration tests for app generator', function (
     var spec = {
       appType: 'scaffold',
       appName: 'todo',
+      web: true,
```

Comment:
Minor: If we create a separate issue for the watson conversation fix mentioned in https://github.com/IBM-Swift/generator-swiftserver/pull/206/files#r117448397 it would be nice to move this fix for the watson conversation test over to it, to keep things together. That said, minor issue should not block merging this.